TwitchChatVotingProxy/VotingReceiver/YTapiV1VotingReceiver.cs

```diff
@@ -0,0 +1,276 @@
+using Shared;
+using Serilog;
+using System;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using System.Text;
+using System.Net.Http;
+using System.Text.Json.Nodes;
+
+namespace TwitchChatVotingProxy.VotingReceiver
+{
+    internal class YTapiV1VotingReceiver : IVotingReceiver
+    {
+
+        public event EventHandler<OnMessageArgs>? OnMessage;
+
+        private static ILogger logger = Log.Logger.ForContext<YouTubeVotingReceiver>();
+        private Magic details;
+        private class Magic
+        {
+            private static HttpClient client = new HttpClient();
+            private static string? continuation = null;
+            private string channelId;
+            private string videoId;
+            public Magic(string cID, string vID)
+            {
+                client.DefaultRequestHeaders.Add(
+                    "User-Agent", "C# dotnet 6"
+                );
+                channelId = cID;
+                videoId = vID;
+            }
+            public static byte[] vn(long val)
+            {
+                if (val < 0)
+                {
+                    throw new Exception("ValueError");
+                }
+                int bufSize = (int)(Math.Log2(val + 1) / 7) + 1;
+                byte[] buf = new byte[bufSize];
+                int index = 0;
+                while ((val >> 7) > 0)
+                {
+                    byte m = (byte)(val & 0xFF | 0x80);
+                    buf[index++] = m;
+                    val >>= 7;
+                    if (index >= bufSize)
+                    {
+                        throw new Exception("index was larger than the buffer size. This shouldn't happen");
+                    }
+                }
+                buf[index] = (byte)val;
+                return buf;
+            }
+            public static byte[] concat(byte[] one, byte[] two)

```

Comment:
I think we should use LINQ like described in this answer instead of creating a custom implementation https://stackoverflow.com/a/59262/10315665?

Reply:
VS Code shows me an error when I try to use the Concat method. Maybe I'll just use the overload which takes a list of `byte[]`.